### PR TITLE
Disambiguate faculty/staff and students

### PR DESCRIPTION
**Pull request by fatlotus**
Kinda dumb but — it seems Medical Center staff people are deterred by the "Hospital Staff" part.

### CODE REVIEW

**Comment by georgeteo** [comment]
OCHA not OSHA?

**Comment by fatlotus** [comment]
Eventually, yes — but that'll require changing a bunch of logic :(

File: caravel/model/affiliation.py [hunk 1]
```diff
@@ -51,9 +51,10 @@ class AffiliationMixin(ndb.Model):
     _aff = None
 
     AFFILIATIONS = [
-        ("bsd_hospital", "BSD/Hospital Staff"),
+        ("bsd_hospital", "BSD/Hospital"),
+        ("bfi_booth", "BFI/Booth"),
         ("osha_faculty_staff", "University Faculty/Staff"),
```
Comment (georgeteo) [inline caravel/model/affiliation.py]: Suggestion: University (Non-BSD) Faculty/Staff

File: caravel/model/affiliation.py [hunk 2]
```diff
@@ -51,9 +51,10 @@ class AffiliationMixin(ndb.Model):
...
-        ("osha_current_student", "Current Student"),
+        ("osha_current_student", "University Student"),
```
Comment (georgeteo) [inline caravel/model/affiliation.py]: Insert "(Non-BSD)"?

File: caravel/model/affiliation.py (hunk 1 above)
Comment (georgeteo) [inline caravel/model/affiliation.py]: Or Just (Non-BSD) Faculty/Staff.

My point is to try to make the two sets as disjoint as possible. University is a super set of BSD/Hospital.

**Comment by fatlotus** [comment]
Bump!

**Pull request merged**